src/univariate/continuous/normal.jl

```diff
@@ -74,22 +73,21 @@ kurtosis(d::Normal{T}) where {T<:Real} = zero(T)
 
 entropy(d::Normal) = (log2π + 1)/2 + log(d.σ)
 
-
 #### Evaluation
-
-@_delegate_statsfuns Normal norm μ σ
-
 gradlogpdf(d::Normal, x::Real) = (d.μ - x) / d.σ^2
+logpdf(d::Normal, x::Real) = -(x - d.μ)^2 / 2 / d.σ^2 - log(2π * d.σ^2) / 2
+pdf(d::Normal, x::Real) = exp(logpdf(d, x))
+cdf(d::Normal, x::Real) = (1 + erf((x - d.μ) / d.σ / √2)) / 2
+logcdf(d::Normal, x::Real) = log1p(erf((x - d.μ) / d.σ / √2)) - logtwo
```

Comment:
Similarly, look at
https://github.com/JuliaStats/StatsFuns.jl/blob/316cebb69d4e28e6a4b17d72eee112e61c3b9b77/src/distrs/norm.jl#L66-L68

Reply:
@Nosferican comment to address here